### PR TITLE
add deprecation notice

### PR DESCRIPTION
**Pull request by Rich-Harris**
**tl;dr: I'm proposing that this project be considered inactive, in favour of Rollup and Babel. This PR contains 0.7.5, with the deprecation notice plus a couple of recent fixes**

Version 0.8 of Esperanto will use [Rollup](https://github.com/rollup/rollup) for bundling, rather than Esperanto itself (see #184). People who want the most up-to-date version of Rollup should use Rollup directly, rather than via Esperanto.

Which leaves the question of what this project is *for*. Esperanto can do one-to-one module transformations, but my guess is that the number of people who need to do that but aren't already using [Babel](https://babeljs.io/) is rather small. Since Babel can also transform ES6 modules to AMD, CommonJS and UMD (and SystemJS!), there's really no need to use both.

Since I intend to continue working on Rollup, and think it has a brighter future than Esperanto, I think this project should basically be abandoned.

TODO:

* [x] Improve Rollup documentation (including a 'migrating from Esperanto' page?)
* [ ] Create a Rollup website?

Thoughts? @eventualbuddha / @stefanpenner / anyone

### CODE REVIEW

**Comment by stefanpenner** [comment]
thanks for the headsup, we have our transition plan!

**Comment by jmeas** [comment]
:cry: c u esperanto
